src/vivarium/framework/values/pipeline.py

```diff
@@ -186,6 +186,9 @@ def _call(self, *args: Any, skip_post_processor: bool = False, **kwargs: Any) ->
     def __repr__(self) -> str:
         return f"_Pipeline({self.name})"
 
+    def __hash__(self) -> int:
```

Comment:
What's this going to be used for?

Reply:
This makes sure pipelines remain hashable, which they were previously. Since pipelines are guaranteed to have unique names in a given simulation, this seems sufficient to disambiguate for a hash function. Pipelines not being hashable breaks some VPH components which want to put pipelines into a set.